FLIP2PDF/Acknowledgements.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en-US", xmlns="http://www.w3.org/1999/xhtml">
   <head>
 <meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
-    <title>Flip2PDF Software Acknowledgements</title>
+    <title>FLIP2PDF Software Acknowledgements</title>
```

Comment:
There's also a header on line 19 with the same text

Reply:
fixed